content/200-concepts/100-components/02-prisma-client/058-transactions.mdx

```diff
@@ -302,6 +302,73 @@ transaction functions. We recommend you get in and out as quick as possible!
 
 </Admonition>
 
+### Transaction timing issues
+
+<Admonition type = "note">
+
+- The solution in this section does not apply to MongoDB, because MongoDB does not support [isolation levels](https://www.prisma.io/dataguide/intro/database-glossary#isolation-levels).
+- The timing issues discussed in this section do not apply to CockroachDB and SQLite, because these databases only support the highest `Serializable` isolation level.
+
+</Admonition>
+
+When two or more transactions run concurrently in certain isolation levels, timing issues can cause conflicts, such as the violation of unique constraints. For example, consider the following sequence of events where Transaction A and Tranasaction B both attempt to execute a `deleteMany` then a `createMany` operation:
+
+1. Transaction A: `deleteMany` removes all rows that would cause a conflict.
+1. Transaction B: `deleteMany` operation.
+1. Transaction B: `createMany` operation creates a new set of rows.
+1. Transaction B: Transaction B is committed.
+1. Transaction A: `createMany` operation. The newly-inserted rows conflict with the rows that Transaction B committed at step 4.
+
+In PostgreSQL and Microsoft SQL Server, the default [isolation level](https://www.prisma.io/dataguide/intro/database-glossary#isolation-levels) is `ReadCommited`. This isolation level does not guarantee repeatable reads. If your application commits another transaction between a `deleteMany` and a `createMany` call, then the current transaction might not observe the newly created rows.
+
+To avoid this error, you can set higher isolation levels of either `RepeatableRead` or `Serializable`. From Prisma version 4.4.0, you do not need to set the isolation level in your database configuration. You can instead set a higher isolation level on the transactions.
```

Comment:
```suggestion
To avoid this problem, you can set higher isolation levels of either `RepeatableRead` or `Serializable`. From Prisma version 4.4.0, you do not need to set the isolation level in your database configuration. You can instead set a higher isolation level on the transactions.
```